doc/source/train/user-guides/using-gpus.rst

```diff
@@ -8,22 +8,13 @@ be done in a few lines of code.
 The main interface for configuring scale and resources
 is the :class:`~ray.train.ScalingConfig`.
 
-Scaling Configurations in Train (``ScalingConfig``)
----------------------------------------------------
+Overview
+--------
+Ray Train's :class:`~ray.train.ScalingConfig` configures the number of training workers
+and the resources they should use.
 
-The scaling configuration specifies distributed training properties like the number of workers or the
-resources per worker.
-
-The properties of the scaling configuration are :ref:`tunable <tune-search-space-tutorial>`.
-
-.. literalinclude:: ../doc_code/key_concepts.py
-    :language: python
-    :start-after: __scaling_config_start__
-    :end-before: __scaling_config_end__
-
-.. seealso::
-
-    See the :class:`~ray.train.ScalingConfig` API reference.
+A *worker* here refers to a *training worker*. Technically, it's a
+:ref:`Ray Actor <actor-key-concept>` that runs your training function.
```

Comment:
Haha now that this is so concise I think should actually just get rid of the section header and combine it with the intro!

Something like this:
```
Increasing the scale of a Ray Train training run is simple and can be done in a few lines of code.
The main interface for this is the :class:`~ray.train.ScalingConfig`, 
which configures the number of workers and the resources they should use.

In this guide, a *worker*  refers to a Ray Train distributing training worker,
which is a :ref:`Ray Actor <actor-key-concept>` that runs your training function.

Reply:
I ended up updating this again because I felt it helped with readability Main things were that I thought the first 3 sentences could be combined into two, and that the ordering between how we use "training worker" and then define "worker" was a little odd.